src/utils/callbacks.js

```diff
@@ -63,7 +65,7 @@ const Callbacks = () => {
    */
   const trigger = (name, ...args) => {
     if (callbacks[name]) return callbacks[name](...args);
-    return () => {};
+    return () => undefined;
```

Comment:
Is not better to throw an error here? if the callbackname is wrong you don't want this to keep working right?

Reply:
The thing is that checking throwing and error is a little bit aggressive imo.
A noop is a little bit more subtle in that regard, at most returning a message saying that the specified callback is not supported and will never be triggered is a better way of managing this.

What do you think?